.travis.yml

```diff
@@ -1,9 +1,20 @@
 language: go
-go:
-  - 1.5
+go: 1.5
```

Comment:
Note to self: undo the change to this line. Will make a cleaner merge with other PR later.